client/src/components/App.js

```diff
@@ -1,11 +1,29 @@
 import React from 'react'
+import { connect } from 'react-redux'
+import { logout } from '../actions'
 
-function App() {
-  return (
-    <div>
-      <p>app!</p>
-    </div>
-  )
+class App extends React.Component {
+  onClickLogout = () => this.props.logout()
+
+  render() {
+    return (
+      <div>
+        <p>stuff soon....</p>
+        <p to="/" onClick={this.onClickLogout}>
+          Logout
```

Comment:
Setup localization support right away, as it is harder to do so later on.
https://github.com/i18next/react-i18next